webapi/src/main/scala/org/knora/webapi/responders/v1/ProjectsResponderV1.scala

```diff
@@ -276,17 +266,19 @@ class ProjectsResponderV1 extends ResponderV1 {
     /**
       * Gets the members of a project with the given IRI.
       *
-      * @param projectIri the IRI of the project.
+      * @param projectIri    the IRI of the project.
       * @param userProfileV1 the profile of the user that is making the request.
       * @return the members of a project as a [[ProjectMembersGetResponseV1]]
       */
     private def projectMembersByIRIGetRequestV1(projectIri: IRI, userProfileV1: UserProfileV1): Future[ProjectMembersGetResponseV1] = {
 
-        log.debug(s"projectMembersByIRIGetRequestV1 - projectIRI: $projectIri")
+        //log.debug("projectMembersByIRIGetRequestV1 - projectIRI: {}", projectIri)
 
+        for {
+            projectExists: Boolean <- projectByIriExists(projectIri)
 
+            _ = if (!projectExists) throw NotFoundException(s"Project '$projectIri' not found.")
```

Comment:
At the moment we don't delete users/projects/groups but set their `status` as `false`, so that this check is still valid in the case of a "deleted" project.